crates/bevy_dev_tools/src/lib.rs

```diff
@@ -0,0 +1,35 @@
+#![warn(missing_docs)]
+//! This crate provides additional utilities for the [Bevy game engine](https://bevyengine.org),
+//! focused on improving developer experience.
+
+use bevy_app::prelude::*;
+#[cfg(feature = "bevy_ci_testing")]
+pub mod ci_testing;
+
+/// Enables developer tools in an [`App`]. This plugin is added automatically with `bevy_dev_tools`
+/// feature.
+///
+/// Warning: It is not recommended to enable this in a release build.
+///
+/// To enable developer tools, you can either:
```

Comment:
There's a third, better way to do this.

Create your own crate feature e.g. `dev_mode`, which enables the `bevy_dev_tools` dependency (and any other dev tools you might have).